src/js/hca/utils/validations.js

```diff
@@ -88,7 +88,11 @@ function isValidDate(day, month, year) {
 }
 
 function isValidName(value) {
-  return /^[a-zA-Z][a-zA-Z '\-]*$/.test(value);
+  return /^[a-zA-Z '\-]*$/.test(value);
+}
+
+function isValidMinimumName(value) {
```

Comment:
Is it possible to get rid of the other `isValidName` and call this function `isValidName`? These are HCA specific validation functions.